package.json

```diff
@@ -28,6 +28,6 @@
     "typescript": "^4.5.2"
   },
   "dependencies": {
-    "lit": "^2.0.2"
+    "lit": "2.4.1"
```

Comment:
Pegged to 2.4.1 to match slickstream-client & guild-web. The version diff was causing issues w/ my local dev environment